source/_components/plugwise.markdown

```diff
@@ -0,0 +1,147 @@
+---
+title: "Plugwise"
+description: "Plugwise Climate integration."
+logo: plugwise.png
+ha_category: Climate
+ha_iot_class: Local Polling
+ha_release: 0.97
+---
+
+This enables [Plugwise](https://plugwise.com) [Anna](https://www.plugwise.com/en_US/products/anna) thermostats to be integrated. This integration talks locally to your **Smile** interface, you will need it's secret key and IP address.
+
+To find your IP address using the Plugwise App. When connected choose the 'Settings'-icon and go to 'HTML-interface'. Go to the (now little lower) 'Settings'-icon and choose 'Preferences'. Under 'System' then 'Networking' you will find your IP address.
+
+### Configuration
+
+```yaml
+# Minimal configuration.yaml entry
+climate:
+  - platform: plugwise
+    password: your_short_id # required, the ID on the smile (some string of 6 characters)
+    host: local_ip_address  # required, the IP-address of your smile
+```
+
+```yaml
+# Example configuration.yaml entry to show only devices at your vacation and primary homes
```

Comment:
Maybe move this example below the configuration variables with a `Full example` title?

Reply:
I've commited changes (and did the same with the service calls)

Reply:
I see the full example still above the `configuration variables`

Reply:
Marking unresolved, my apologies.

Moved it accordingly now.